app/serializers/generic_event_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class GenericEventSerializer < ActiveModel::Serializer
+  type 'generic_events'
```

Comment:
do we want to expose this to clients as `generic_events` or `events` 🤔 
Internally its ok to be `generic_events` but I'm wondering around how we surface this

Reply:
No we don't. Fixed